File: app/src/main/java/com/wjxls/mmkvapplication/MainActivity.java
package com.wjxls.mmkvapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;

import com.wjxls.mmkvlibrary.WJSharePreferenceConfigImpl;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        WJSharePreferenceConfigImpl.getWjSharePreferenceConfigImpl().setString("name","张三");
        Log.e("MainActivity",WJSharePreferenceConfigImpl.getWjSharePreferenceConfigImpl().getString("name",""));
    }
}

File: app/src/main/java/com/gxx/mmkvapplication/MainActivity.kt
package com.gxx.mmkvapplication

import android.app.Activity
import android.os.Bundle
import android.util.Log
import com.gxx.mmkvlibrary.MMKvUtils
import kotlinx.coroutines.GlobalScope
import kotlinx.coroutines.launch

class MainActivity:Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        MMKvUtils.putString("name", "张三");
        Log.e("MainActivity", "使用mmkv->" + MMKvUtils.getString("name", "")+"????")

        GlobalScope.launch {
            MMKvUtils.writeFileString("abc","你好你好你好你好你好你好你好你好你好1111111111")
            Log.e("MainActivity", MMKvUtils.getFileString("abc"))
        }
    }

}